.idea/misc.xml

```diff
@@ -16,4 +16,4 @@
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
```

Comment:
@mayank-kgp Well, I have tried the following ways but it's still not working:
1) Remove the new line in local copy and push the changes using git
2) Copy the original misc.xml in raw form to my local misc.xml and then push using git
3) Remove the new line manually in github.
4) Download the original copy of misc.xml and upload it at ./.idea/ in the proper branch (of this PR) in my forked repo in github

None of them seem to work. Anyway, if someone is not editing this file and adding it in his PR, there won't be any conflict in merging his copy.